platform-storage.md

```diff
@@ -1,82 +1,84 @@
 ---
 title: Platform storage
 
-description:
+description: Describes the different storage technologies used on the OpenShift platform.
 
 keywords: netapp, net app, nfs, storage, pvc, persistent, storage options
 
-page purpose:
+page purpose: Describes different platform technologies, types, and services and their details.
 
-audience:
----
+audience: technical lead, developer
 
-<!-- OM notes:
-This external page is linked from this our page where we list storage services available on the Platform (in this paragraph. The S3 Object store service is provided by OCIO Enterprise Hosting team, not us, and it is recommended for the product team that has a need to store large amount of unstructured data such as documents, images, files, etc.
+author: Jonathan Bond
 
-This page includes more details about who should be using the S3 object store and how to get space there. The only change to the page above would be to remove the reference to medium quota size, as we have recently change the quota model and we do not have medium quota anymore. The doc can just say that teams that need to store large amount of unstructured data should use the OCIO’s S3 Object store service instead of the NetApp storage in Openshift. NetApp storage is very expensive and therefore is not suitable for storing large amounts of data
--->
+content owner: Olena Mitovska
+---
 
 # Platform storage
 
-- [Storage Technologies](#storage-technologies)
-- [Tools](#tools)
-- [Storage Class FAQ](#storage-class-faq)
-
 We use several different storage technologies on the OpenShift platform. Currently, we have three types of persistent storage available:
 
-* **FILE**: File storage is an all-purpose storage type that can be attached to one or more containers. It's the recommended storage class for most general application uses and is powered by NFS.
+* **FILE**: File storage is an all-purpose storage type that can be attached to one or more containers. It's the recommended storage class for most general application uses and is powered by Network File System (NFS).
 
 * **BLOCK**: Block storage offers additional performance and reliability for databases or similar workloads, but is only able to be attached to 1 container at a time. This is powered by iSCSI.
```

Comment:
Heads, up this isn't true! You can attached `netapp-block-standard` to more than one pod if you want to. The ability to mount a PV to multiple pods is set in your config, by specifying if the PV should be Read-Write-Once (RWO) or Read-Write-Many (RWX). The file/block storage classes have nothing to do with this!